File: 217-contains-duplicate/contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        removeDup = set()
        for i in nums:
            if i in removeDup:
                return True
            removeDup.add(i)
        return False
        

File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashMap<Integer, Integer> dup = new HashMap<Integer, Integer>();
        for(int i: nums)
        {
            if(dup.get(i) != null)
            {
                return true;
            }
            dup.put(i, 1);
        }
        return false;
    }
}